hw2/coin_HW2

```diff
@@ -0,0 +1,46 @@
+const calculateChange = (amt) => {
+  //Several Check cases, if the amount is zero, negative or too big
+  if (amt == 0) console.log("Amount cannot be zero.");
+  else if (amt < 0) console.log("Amount cannot be negative.");
+  else if (amt > 10) console.log("Error: the number is too large");
+  else {
+    //Calculate the no of dollars and store it in dollar variable
+    dollar = Math.trunc(amt / 1);
+    remainingquarters = amt - dollar;
+
+    //Calculate the no of Quarters and store it in quarter variable
+    quartes = Math.trunc(remainingquarters / 0.25);
+    remainingdime = remainingquarters - quartes * 0.25;
+
+    //Calculate the no of Dime and store it in dime variable
+    dime = Math.trunc(remainingdime / 0.1);
+    remainingnickles = remainingdime - dime * 0.1;
+
+    //Calculate the no of Nickel and store it in nickel variable
+    nickel = Math.trunc(remainingnickles / 0.05);
+    remainingpennies = remainingnickles - nickel * 0.05;
+
+    //Print the total number of denominations
+    pennies = Math.trunc(remainingpennies / 0.01);
+    console.log(
```

Comment:
Maybe it is also a good idea to consider singular and plural on the result output